src/olympia/api/serializers.py

```diff
@@ -40,9 +41,16 @@ def get_fields(self):
                 fields[key] = ESTranslationSerializerField(source=field.source)
         return fields
 
-    def to_native(self, data):
+    def to_internal_value(self, data):
         obj = self.fake_object(data)
-        return super(BaseESSerializer, self).to_native(obj)
+        return super(BaseESSerializer, self).to_internal_value(obj)
+
+    def to_representation(self, data):
+        if isinstance(data, Result):
+            data = data.to_dict()
```

Comment:
Not sure why you made this change (and the related attributes -> dictionary items changes) ?